pkg/gds/gds_test.go

```diff
@@ -0,0 +1,35 @@
+package gds_test
+
+import (
+	"context"
+	"time"
+
+	api "github.com/trisacrypto/trisa/pkg/trisa/gds/api/v1beta1"
+)
+
+// TestStatus tests that the Status RPC returns the correct status response.
+func (s *gdsTestSuite) TestStatus() {
+	s.LoadEmptyFixtures()
```

Comment:
```suggestion
```

Since this test doesn't require the database at all, it's better to not specify any fixtures - e.g. the test is completely agnostic to the fixture. That will minimize the switching the test package does.